server/routes/induction/create/workedBeforeCreateController.test.ts

```diff
@@ -200,68 +200,74 @@ describe('workedBeforeCreateController', () => {
       expect(res.redirect).toHaveBeenCalledWith('/prisoners/A1234BC/create-induction/previous-work-experience')
       expect(req.session.workedBeforeForm).toBeUndefined()
       const updatedInduction = req.session.inductionDto
-      expect(updatedInduction.previousWorkExperiences.hasWorkedBefore).toEqual(true)
+      expect(updatedInduction.previousWorkExperiences.hasWorkedBefore).toEqual('YES')
     })
 
-    it('should update InductionDto and display Personal Skills page given form is submitted with worked before NO', async () => {
-      // Given
-      const inductionDto = aLongQuestionSetInductionDto()
-      inductionDto.previousWorkExperiences = undefined
-      req.session.inductionDto = inductionDto
-
-      const workedBeforeForm: WorkedBeforeForm = {
-        hasWorkedBefore: YesNoValue.NO,
-      }
-      req.body = workedBeforeForm
-      req.session.workedBeforeForm = undefined
-
-      // When
-      await controller.submitWorkedBeforeForm(
-        req as undefined as Request,
-        res as undefined as Response,
-        next as undefined as NextFunction,
-      )
-
-      // Then
-      expect(res.redirect).toHaveBeenCalledWith('/prisoners/A1234BC/create-induction/skills')
-      expect(req.session.workedBeforeForm).toBeUndefined()
-      const updatedInduction = req.session.inductionDto
-      expect(updatedInduction.previousWorkExperiences.hasWorkedBefore).toEqual(false)
-    })
-
-    it('should update inductionDto and redirect to Check Your Answers given previous page was Check Your Answers and worked before is changed to No', async () => {
-      // Given
-      const inductionDto = aLongQuestionSetInductionDto()
-      req.session.inductionDto = inductionDto
-
-      const workedBeforeForm: WorkedBeforeForm = {
-        hasWorkedBefore: YesNoValue.NO,
-      }
-      req.body = workedBeforeForm
-      req.session.workedBeforeForm = undefined
-
-      req.session.pageFlowHistory = {
-        pageUrls: [
-          '/prisoners/A1234BC/create-induction/check-your-answers',
-          '/prisoners/A1234BC/create-induction/has-worked-before',
-        ],
-        currentPageIndex: 1,
-      }
-
-      // When
-      await controller.submitWorkedBeforeForm(
-        req as undefined as Request,
-        res as undefined as Response,
-        next as undefined as NextFunction,
-      )
-
-      // Then
-      const updatedInduction = req.session.inductionDto
-      expect(updatedInduction.previousWorkExperiences.hasWorkedBefore).toEqual(false)
-      expect(updatedInduction.previousWorkExperiences.experiences).toEqual([])
-      expect(res.redirect).toHaveBeenCalledWith('/prisoners/A1234BC/create-induction/check-your-answers')
-      expect(req.session.workedBeforeForm).toBeUndefined()
-    })
+    it.each([HasWorkedBeforeValue.NO, HasWorkedBeforeValue.NOT_RELEVANT])(
```

Comment:
Much love for `it.each` 😁 👍 
(In some early test code you'll find we create and loop through an array - we didn't know about `it.each` when we wrote those older tests; this is a much neater way 👍 )